src/components/MoviesList/MoviesList.tsx

```diff
@@ -1,6 +1,24 @@
 import React from 'react';
 import './MoviesList.scss';
+import { Movies } from '../types/Movies';
+import { MovieCard } from '../MovieCard';
 
-export const MoviesList: React.FC = () => (
-  <>Put the list here</>
+type Props = {
+  movies: Movies[];
+};
+
+export const MoviesList: React.FC<Props> = ({ movies }) => (
+  <div className="movies">
+    {movies.map(({
+      title, description, imgUrl, imdbUrl, imdbId,
+    }) => (
+      <MovieCard
+        title={title}
+        description={description}
+        imgUrl={imgUrl}
+        imdbUrl={imdbUrl}
+        imdbId={imdbId}
+      />
+    ))}
```

Comment:
Add key for mapped element:
```suggestion
    {movies.map(({
      title, description, imgUrl, imdbUrl, imdbId,
    }) => (
      <MovieCard
        key=...
        title={title}
        description={description}
        imgUrl={imgUrl}
        imdbUrl={imdbUrl}
        imdbId={imdbId}
      />
    ))}
```